app/views/candidate_mailer/confirm_sign_in.text.erb

```diff
@@ -0,0 +1,12 @@
+Dear <%= @candidate_user.email_address %>
```

Comment:
N.B. Due to magic link log in we  may only have access to candidate email at this stage, so I defaulted to this in case account does not yet hav a a name provided.

Reply:
Pretty sure we can get rid of this line altogether. Whatcha think @Tomcowan1?

Reply:
Hello - I'll take a look through the content and come back with edits. One thing that does come to mind is whether people will know their devices by IP addresses? Is it possible to base it on something like location instead?

Reply:
Thanks Tom, 

This email will only fire if there is a log in to an account from a device with different credentials. If a user logs in say from their phone it should be fairly obvious that this is the case as their IP address will stay the same (provided they're using the same internet) but the listed browser will change.

Admittedly, a location would make things slightly clearer. I have just had a look into whether we can access this from the same request or work it out using Geocoder but unfortunately my knowledge runs out here. @malcolmbaig would it be possible to pull through location here?

If it is possible we can amend this update this or update a future PR, depending on the complexity.

Reply:
You can get a location from IP address… chances that it’s right however, which could then make people even more concerned when they needn’t be.

Reply:
OK. I was probably channelling the Apple ones where it's clear that you haven't just logged in because it's showing you a completely different country.

Reply:
Cool well we can park it for now and if we get some support calls about it we can always address it then :) Open to something being carded up too.